### DISCUSSION TITLE
Failed to load model after downloading

### DISCUSSION THREAD

**Discussion by Helmermat** [oobabooga/text-generation-webui#2982] [Q&A]
I tried oobabooga for the first time and I downloaded a model via the UI.
After inserting the Huggingface link, the download started and completed successfully.
Then I selected ExLlama_HF as model loader and refreshed the dropdown menu containing the available models. The new model I just downloaded was displayed in that menu and I clicked on it to load it.

Immediately after that I got an error message on my windows console prompt.
ERROR: Failed to load the model.

FileNotFoundError: [Errno2] No such file or directory: 'models\\notstoic_pygmalion-13b-4bit-128g\\pytorch_model-00001-of-00003.bin'

What did I do wrong?

**Comment by wp337** [comment]
Start by turning off any VPNs or proxies. Then start the model on the command line, and then enter the WEBUI to try it out
cd C:\Users\Administrator\text-generation-webui
python server.py --model chatglm-6b --listen --trust-remote-code

**Comment by Helmermat** [comment]
E:\LLM\oobabooga_windows\text-generation-webui> python server.py --model notstoic_pygmalion-13b-4bit-128g --listen --trust-remote-code
Traceback (most recent call last):
File "E:\LLM\oobabooga_windows\text-generation-webui\server.py", line 12, in <module>
import gradio as gr
ModuleNotFoundError: No module named 'gradio'

**Reply by BelloGoo** [reply]
i got the same issue, im thinking if its about the format of models. i found some .safetensors models and they could be loaded, but .bin.
how does it go for now?

**Comment by wp337** [comment]
邮件已收到，如需及时联系可致电13521993276QQ:93793234   微信：93793234

**Comment by wp337** [comment]
邮件已收到，如需及时联系可致电13521993276QQ:93793234   微信：93793234
